test/unit/services/domain/test-workload-points.js

```diff
@@ -77,6 +78,7 @@ describe('services/domain/workload-points', function () {
     expect(workloadPoints.weightingArmsCommunity).to.equal('10.1')
     expect(workloadPoints.weightingArmsLicense).to.equal('10.2')
     expect(workloadPoints.weightingWarrants).to.equal('0.0')
+    expect(workloadPoints.isT2A).to.equal('false')
   })
 
   it('should raise a ValidationError if a Community Weighting field is out of range', function () {
```

Comment:
Could we add a unit test to test if the isT2A flag is set?

Reply:
👍